src/ui/src/js/controllers/job/import_systems.js

```diff
@@ -0,0 +1,99 @@
+import template from '../../../templates/import_systems.html'
+
+jobImportSystemsController.$inject = ['$scope', '$rootScope', '$uibModal', 'JobService'];
+
+/**
+ * jobImportSystemsController - Controller for job import.
+ * @param  {Object} $scope            Angular's $scope object.
+ * @param  {Object} $rootScope        Angular's $rootScope object.
+ * @param  {Object} $uibModal         Angular UI's $uibModal object.
+ * @param  {Object} JobService        Beer-Garden's job service.
+ */
+export function jobImportSystemsController( $scope, $rootScope, $uibModal, JobService) {
+  $scope.setWindowTitle('Import');
+
+  $scope.response = $rootScope.sysResponse;
+  $scope.data = $rootScope.systems;
+
+  $scope.openImportSystemsPopup = function () {
+      let popupInstance = $uibModal.open({
+          animation: true,
+          controller: 'JobImportModalController',
+          template: template
+      });
+
+      popupInstance.result.then(
+          function(resolvedResponse){
+            let jsonFileContents = resolvedResponse.jsonFileContents;
+            console.log(jsonFileContents);
+            JobService.importJobs(jsonFileContents).then(
+                function(response) {
+                    alert("Successful import. Refresh page to see imported Jobs.")
```

Comment:
This is a better option. I'll look into what's necessary to make that happen and if it's not too hard, will implement. Having messages, errors, etc., reflected on the requests page was an initial design, but it would have involved changing the parent page and its associated controllers, etc, in ways that seemed certain to introduce problems. Hence the decision to create a modal popup. But a simple page refresh might not be so intrusive.